commands/wombat.js

```diff
@@ -0,0 +1,32 @@
+const { SlashCommandBuilder } = require('@discordjs/builders')
+const { reply } = require('../helpers/interactionHelper')
+const { WOMBATS_DIR_NAME } = require('../helpers/constants')
+const path = require('path')
+const fs = require('fs/promises')
+
+const data = new SlashCommandBuilder()
+  .setName('wombat')
+  .setDescription('Post een willekeurige foto van een wombat.')
+
+async function execute (interaction) {
+  let wombatList
+  try {
+    wombatList = await fs.readdir(path.resolve(WOMBATS_DIR_NAME))
+
+    if (wombatList.length === 0) {
+      throw new Error("Map met foto's is leeg")
+    }
+  } catch (error) {
```

Comment:
Is this error automatically logged? If not, it might be good idea to include a log statement here

Reply:
It was not, I've added it.